runtime/src/bank.rs

```diff
@@ -4242,24 +4259,26 @@ impl Bank {
             "commit_transactions() working on a bank that is already frozen or is undergoing freezing!"
         );
 
-        self.increment_transaction_count(tx_count);
+        self.increment_transaction_count(committed_transactions_count);
```

Comment:
hmmm, this is good to consider, I was only thinking about stuff that would break snapshots/compatibility (luckily none of these fields are returned by `get_fields_to_serialize()` to be serialized into the snapshot).

Reverted for now here: https://github.com/solana-labs/solana/pull/22802/commits/7d410bea290370c043946e228b840e2269d3536a

Reply:
I think we can just add a feature for it to activate the new count at a certain epoch to keep stats across all nodes consistent

Reply:
added feature here: https://github.com/solana-labs/solana/pull/22802/commits/9f6deee5e9df5740c823b04c4aa5fb23d2609f1e